custom-resources/cert-manager-issuer.tf

```diff
@@ -1,7 +1,11 @@
 locals {
-  dns_zones           = try(var.core_services.azure_dns_zones.dns_zones, [])
-  dns_resource_group  = try(var.core_services.azure_dns_zones.resource_group, "")
-  enable_cert_manager = length(local.dns_zones) > 0 && length(local.dns_resource_group) > 0
+  enable_azure_dns = length(var.core_services.azure_dns_zones) > 0
```

Comment:
`var.core_services.azure_dns_zones` is a map not an array so `length` here is not what we're looking for. I believe you want `length(var.core_services.azure_dns_zones.dns_zone) > 0`

Reply:
Agreed, I'll fix it.